publish_all.py

```diff
@@ -0,0 +1,27 @@
+import pathlib
+import subprocess
+
+all_paths = pathlib.Path(".").glob("**/pyproject.toml")
+
```

Comment:
Do we need to add the top level pyproject.toml to all_paths?

Reply:
Nope! But I did just push an update that will use pathing relative to the publish_all.py file just in case someone tries to run this from a different directory and it has unexpected outcomes.